package.json

```diff
@@ -23,7 +23,6 @@
     "mqlight": "^1.0.0"
   },
   "devDependencies": {
-    "jscs": "^2.8.0",
     "eslint": "^1.10.3",
     "eslint-config-strongloop": "^1.0.2",
```

Comment:
I upgraded this connector at https://github.com/strongloop/loopback-connector-mqlight/pull/13